CMakeLists.txt

```diff
@@ -126,13 +126,38 @@ if (NOT DEFINED XEUS_CPP_KERNELSPEC_PATH)
     set(XEUS_CPP_KERNELSPEC_PATH "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR}/")
 endif ()
 
-set(XEUS_CPP_PATH "$ENV{PATH}")
-set(XEUS_CPP_LD_LIBRARY_PATH "$ENV{LD_LIBRARY_PATH}")
-set(XEUS_CPP_RESOURCE_DIR ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/clang/${CppInterOp_CLANG_VERSION})
-configure_file (
-    "${CMAKE_CURRENT_SOURCE_DIR}/share/jupyter/kernels/xcpp/kernel.json.in"
-    "${CMAKE_CURRENT_BINARY_DIR}/share/jupyter/kernels/xcpp/kernel.json"
-)
+function(configure_kernel kernel)
```

Comment:
This function is mixing configuration and install. This should be splitted:
- The different `kernel.json.in` files must be configured to produce `kernel.json` files in the source directory
- Another function should be implemented to copy the different files to the binary dir (in the install section of the CMakeLists.txt)

Reply:
Why we should produce the `kernel.json` in the source directory? The build process should not touch the source directories. That's the recommended way of cmake but in addition the source directory can be on read-only file system. What do we gain if we output the `kernel.json` in the source directory?

Reply:
This was for consistency with the cmake files of other xeus projects; but you're right, we should not modify the source directory.

Reply:
Ok, understood. Thanks for reviewing!